JSON_for_IO-Link.yaml

```diff
@@ -4135,8 +4114,6 @@ components:
               default: 0
         blockList:
           type: object
-          required:
```

Comment:
I removed the required option in order to get the linter passed, but we should think about if all property entries are required. 
If they are really required, they must be listed one by one here:
`required:
     - '2402'
     - '2403'
     - ...
 `